checker/tests/nullness/Issue2407.java

```diff
@@ -0,0 +1,32 @@
+import org.checkerframework.checker.nullness.qual.EnsuresNonNull;
+import org.checkerframework.checker.nullness.qual.EnsuresNonNullIf;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.RequiresNonNull;
+
+class Issue2407 {
+
+    @NonNull String message = "default string";
```

Comment:
Why the explicit NonNull?
Isn't the interesting thing about this test case that the parameter to `setMessage` is NonNull and the pre-condition is redundant, but still causes issues? So I would remove the NonNull here and document the more interesting aspect of the test case.